src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    public static final SessionFactory inctance = initSessionFactory();
+
+    public HibernateUtil() {
```

Comment:
constructor should be private, which is a good practice for Util classes with static-only methods, so that we could not create an instance of it because there is no need for it